cfgov/jinja2/v1/_includes/organisms/header.html

```diff
@@ -102,6 +96,12 @@
             </a>
             {% endif %}
 
+            {% import 'molecules/global-header-cta.html' as global_header_cta with context %}
+            {{ global_header_cta.render( true, language ) }}
+
+            {% import 'molecules/global-search.html' as global_search with context %}
+            {{ global_search.render() }}
+
```

Comment:
Unrelated to the mm, I moved the search form and CTA molecule below the logo to fix the tab order of the page. Moving them doesn't affect the header's layout because they're both floated right.